pages/common/s.md

```diff
@@ -0,0 +1,19 @@
+# s
+
+> Web search from the terminal.
+
+- Search for query on Google(default provider):
+
+`s {{query}}`
+
+- Search for a query on provider:
+
+`s --provider {{provider}} {{query}}`
+
+- List all providers:
```

Comment:
I think this should be above "search with a given provider". Because first, a person needs to list the providers to get to know what all providers are there to search with something other than google.